src/zcl_abapgit_repo_srv.clas.abap

```diff
@@ -388,7 +388,12 @@ CLASS ZCL_ABAPGIT_REPO_SRV IMPLEMENTATION.
   METHOD zif_abapgit_repo_srv~validate_package.
 
     DATA: lv_as4user TYPE tdevc-as4user,
-          lt_repos   TYPE zif_abapgit_persistence=>tt_repo.
+          lt_repos   TYPE zif_abapgit_persistence=>tt_repo,
+          lv_name    TYPE zif_abapgit_persistence=>ty_local_settings-display_name,
+          lv_owner    TYPE zif_abapgit_persistence=>ty_local_settings-display_name.
```

Comment:
TYPE not aligned horizontally, please run pretty printer

Reply:
My apologies for these stupid typos. How comes abaplint didn't notice these style violations?